CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ cmake_minimum_required(VERSION 3.18)
 
 project(CARE
         LANGUAGES CXX
-        VERSION 0.11.1)
+        VERSION 0.12.0)
```

Comment:
Should this be 0.13?

Reply:
0.11.3.m was the last version in our fork.